test-suite/shared/src/test/require-jdk8/org/scalajs/testsuite/javalib/time/TemporalAccessorTest.scala

```diff
@@ -7,8 +7,8 @@ import org.junit.Test
 import org.junit.Assert._
 import org.scalajs.testsuite.utils.AssertThrows._
 
-abstract class TemporalAccessorTest {
-  val samples: Seq[TemporalAccessor]
+abstract class TemporalAccessorTest[A <: TemporalAccessor] {
+  val samples: Seq[A]
```

Comment:
I think that adding this `A <: TemporalAccessor` is overkill. You could just use `val samples: Seq[TemporalAccessor]` and then make it more specific in each implementation (as `Seq` is covariant).